### PR TITLE
Use V5 app endpoints for V5 clusters.

### PR DESCRIPTION
**Pull request by oponder**
Builds on: https://github.com/giantswarm/happa/pull/909
Towards: https://github.com/giantswarm/giantswarm/issues/8170

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,67 +1,90 @@
-<!doctype html>
+<!DOCTYPE html>
```
Comment (oponder) [inline src/index.html]: this isn't actually in the commit.. I think github is caching the diff in a weird way.

File: src/actions/appActions.js [hunk 2]
```diff
@@ -11,29 +13,26 @@ export function selectCluster(clusterID) {
...
+  console.log('loading1');
   return function(dispatch, getState) {
-    // This method is going to work for NP clusters, now in local dev it is not
-    // working, so early return if the cluster is a NP one.
+    const appsApi = new GiantSwarm.AppsApi();
+
     const nodePoolsClusters = getState().entities.clusters.nodePoolsClusters;
     const isNodePoolsCluster = nodePoolsClusters.includes(clusterId);
+
+    let getClusterAppsFunc;
```
Comment (oponder) [inline src/actions/appActions.js]: Here we decide which api method to call, the v5 or the v4 version, based on whether or not this is a nodepool cluster.

A similar pattern is applied to all the other actions.

File: src/index.html (hunk 1 above)
Comment (oponder) [reply src/index.html]: fixed

File: package.json [hunk 3]
```diff
@@ -47,7 +47,7 @@
     "copy-to-clipboard": "3.2.0",
     "deep-diff": "1.0.2",
     "emotion-theming": "^10.0.10",
-    "giantswarm": "git+https://github.com/giantswarm/giantswarm-js-client.git",
+    "giantswarm": "git+https://github.com/giantswarm/giantswarm-js-client.git#add-v5-app-endpoints",
```
Comment (ograu) [inline package.json]: I guess this is needed by now

**Pull request merged**